src/main/java/hudson/plugins/active_directory/ActiveDirectorySecurityRealm.java

```diff
@@ -195,6 +197,11 @@
      */
     protected List<EnvironmentProperty> environmentProperties;
 
+    /**
+     * If not null, we check if this header exists and use the username in the header to load the user.
+     */
+    protected String userFromHTTPHeader;
```

Comment:
This should probably be named `userFromHttpHeader` to be consistent with the other initialism-based field/property name in this class.